### ISSUE TITLE
List of version numbers should be obtained from jspm

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/prototyper#16]
At the moment MPC uses local, preloaded files

**Comment by Thanood** [comment]
Do you mean `npms` instead of `jspm`?

**Comment by adriatic** [comment]
Of course, thanks for telling me. These two acronyms make me swap them at all times. Let me also comment some more: I created the local .json files, at the time I did not even know about `http://npms.io`, by running ` npm view aurelia-kendoui-bridge > aurelia-kendoui-bridge.txt` in order to get the the file that can easily be converted into a proper .json file. like **[aurelia-kendoui-bridge.json](https://github.com/monterey-framework/prototyper/blob/master/app/src/mpc/data/aurelia-kendoui-bridge.json)**

This approach does not need to stay the same, as the npms API (existing or new that we might ask for if nothing similar exists) might make it simpler. I hope that we could get by with using a single invocation of npms API, where we then internally distribute this data among the various components of the MPC. This way we do not need to worry about caching - assuming of course that the npms API invocation takes a second of the physical time which will still result with very good real time response.

To be precise - there that single npms API call mentioned above takes place **after** selecting the package name in the autocomplete control 😄

**Comment by Thanood** [comment]
Yes, I thought the same. 😃 
I think Monterey should have an `npms` service which MPC will consume. It could evolve from `data/packageService` in this (prototyper) project.
